tests/Validator/UniqueOperationTypesTest.php

```diff
@@ -0,0 +1,463 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests\Validator;
+
+use GraphQL\Type\Schema;
+use GraphQL\Utils\BuildSchema;
+use GraphQL\Validator\Rules\UniqueOperationTypes;
+
+class UniqueOperationTypesTest extends ValidatorTestCase
+{
+    private function expectSDLErrors(string $sdlString, ?Schema $schema, array $errors): void
+    {
+        $this->expectSDLErrorsFromRule(new UniqueOperationTypes(), $sdlString, $schema, $errors);
+    }
+
+    // Validate: Unique operation types
```

Comment:
@simPod recently brought it up, and I am inclined to agree that we should put this in the PHPDoc of the first test of the group in order to have it be sticky in a structural sense.

Reply:
Something like this?
```php
/**
 * @see describe('Validate: Unique operation types')
 * @see it('no schema definition')
 */
```

Reply:
Looks good to me!